lib/containers/container_images.pm

```diff
@@ -129,4 +129,11 @@ sub test_opensuse_based_image {
     assert_script_run("$runtime run --rm refreshed-image sh -c 'zypper -v ref | grep \"All repositories have been refreshed\"'", 120);
 }
 
+sub exec_cmd {
+    my ($image, $runtime, $command, $timeout) = @_;
```

Comment:
just thinking loud. What do you think on having some oo style here.
A `lib/containers/podman.pm` and a `lib/containers/docker.pm` both have the `exec()` but each of them know how to call a `command`.